src/app/components/mnoe-api/admin/teams.svc.coffee

```diff
@@ -0,0 +1,22 @@
+@App.service 'MnoeTeams', ($q, MnoeAdminApiSvc, toastr, MnoErrorsHandler) ->
+
+  teamsApi = (id) ->
```

Comment:
private methods not on the `this` scope usually kept at bottom of file rather than top